docs/BigQueryTable-batchsink.md

```diff
@@ -55,6 +55,9 @@ Should only be used with the Insert operation.
 multiple input records with the same key. For example, if this is set to 'updated_time desc', then if there are
 multiple input records with the same key, the one with the largest value for 'updated_time' will be applied.
 
+**Partition Filter**: Partition filter that can be used for partition elimination during Update operation.
+Should only be used with Update operation for tables where require partition filter is enabled.
```

Comment:
please also add example of what partition filter value should be since its a text box.

Reply:
@CuriousVini we are going to add the following example:

> For example, if the table is partitioned the Partition Filter '_PARTITIONTIME > "2020-01-01" and _PARTITIONTIME < "2020-03-01"', the update operation will be performed in the partitions meeting the criteria.